engine/src/flutter/impeller/tessellator/tessellator_unittests.cc

```diff
@@ -5,82 +5,92 @@
 #include "flutter/testing/testing.h"
 #include "gtest/gtest.h"
 
-#include "flutter/display_list/geometry/dl_path.h"
+#include "flutter/display_list/geometry/dl_path_builder.h"
 #include "impeller/geometry/geometry_asserts.h"
-#include "impeller/geometry/path.h"
-#include "impeller/geometry/path_builder.h"
 #include "impeller/tessellator/tessellator.h"
 #include "impeller/tessellator/tessellator_libtess.h"
 
 namespace impeller {
 namespace testing {
 
-TEST(TessellatorTest, TessellatorBuilderReturnsCorrectResultStatus) {
-  // Zero points.
-  {
-    TessellatorLibtess t;
-    auto path = PathBuilder{}.TakePath(FillType::kOdd);
-    TessellatorLibtess::Result result = t.Tessellate(
-        path, 1.0f,
-        [](const float* vertices, size_t vertices_count,
-           const uint16_t* indices, size_t indices_count) { return true; });
-
-    ASSERT_EQ(result, TessellatorLibtess::Result::kInputError);
-  }
-
-  // One point.
-  {
-    TessellatorLibtess t;
-    auto path = PathBuilder{}.LineTo({0, 0}).TakePath(FillType::kOdd);
-    TessellatorLibtess::Result result = t.Tessellate(
-        path, 1.0f,
-        [](const float* vertices, size_t vertices_count,
-           const uint16_t* indices, size_t indices_count) { return true; });
-
-    ASSERT_EQ(result, TessellatorLibtess::Result::kSuccess);
-  }
-
-  // Two points.
-  {
-    TessellatorLibtess t;
-    auto path = PathBuilder{}.AddLine({0, 0}, {0, 1}).TakePath(FillType::kOdd);
-    TessellatorLibtess::Result result = t.Tessellate(
-        path, 1.0f,
-        [](const float* vertices, size_t vertices_count,
-           const uint16_t* indices, size_t indices_count) { return true; });
-
-    ASSERT_EQ(result, TessellatorLibtess::Result::kSuccess);
-  }
-
-  // Many points.
-  {
-    TessellatorLibtess t;
-    PathBuilder builder;
-    for (int i = 0; i < 1000; i++) {
-      auto coord = i * 1.0f;
-      builder.AddLine({coord, coord}, {coord + 1, coord + 1});
-    }
-    auto path = builder.TakePath(FillType::kOdd);
-    TessellatorLibtess::Result result = t.Tessellate(
-        path, 1.0f,
-        [](const float* vertices, size_t vertices_count,
-           const uint16_t* indices, size_t indices_count) { return true; });
-
-    ASSERT_EQ(result, TessellatorLibtess::Result::kSuccess);
-  }
-
-  // Closure fails.
-  {
-    TessellatorLibtess t;
-    auto path = PathBuilder{}.AddLine({0, 0}, {0, 1}).TakePath(FillType::kOdd);
-    TessellatorLibtess::Result result = t.Tessellate(
-        path, 1.0f,
-        [](const float* vertices, size_t vertices_count,
-           const uint16_t* indices, size_t indices_count) { return false; });
-
-    ASSERT_EQ(result, TessellatorLibtess::Result::kInputError);
-  }
-}
+// TEST(TessellatorTest, TessellatorBuilderReturnsCorrectResultStatus) {
```

Comment:
Is this segment left commented out intentional?

Reply:
Oh, I forgot to mention that I have much of the tessellator stuff commented out while I figure out what its future is.